datalad/metadata/parsers/bids.py

```diff
@@ -145,9 +168,9 @@ def yield_participant_info(fname):
                 val = row[k]
                 if hk is None:
                     hk = 'comment<{}>'.format(normk)
-                elif hk == 'dlp_bids:sex':
-                    val = sex_label_map.get(row[k].lower(), None)
+                if hk == 'comment<sex>':
```

Comment:
btw, why those fields from participant are "comment"? could they be at least somewhat more specific, e.g. `comment<participant:sex>` in case if we are doomed to use a `comment` one

Reply:
Comment is anything that is nowhere defined. The content of these files is pretty much completely unregulated, hence we have no idea what label means what. We can add any prefix though -- it is just a comment.